public/resources/stylesheets/userbar.less

```diff
@@ -477,7 +477,7 @@ a.navbar-login:hover {
 
       li {
         display: block;
```

Comment:
Please remove the ``display: block`` declaration. ``<li>`` items are by definition block-level elements, so we can drop this.